test/standalone.yml

```diff
@@ -3,6 +3,15 @@ services:
 
   action-server:
     image: zenoh-action-server
+    build: ../python/action_server
+    depends_on:
+      - log-forwarder
+      - local-zenoh-router
+    restart: unless-stopped
+
+  action-test:
+    image: zenoh-action-client
+    build: ../python/action_client
     container_name: action
     build: ../rust
```

Comment:
these two lines should be removed. Kindly test your code locally before committing as we don’t have automatic tests setup to catch these mistakes automatically currently